### PR TITLE
Implement local storage

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/Gift/styles.scss [hunk 1]
```diff
@@ -29,4 +32,7 @@
   outline: 0;
   cursor: pointer;
   margin-left: 20px;
+  &:hover {
+    color: black;
```
Comment (mi-chal) [inline src/screens/Gift/styles.scss]: Should be in colors.scss

File: src/store/store.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
 import { createStore } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+
+// import {
+//   getItemFromLocalStorage,
+//   saveItemInLocalStorage,
+// } from 'utils/localStorage';
+// import throttle from 'lodash.throttle';
+
```
Comment (mi-chal) [inline src/store/store.js]: Remove

File: src/store/store.js [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
 import rootReducer from './rootReducer';
 
+// const persistedState = getItemFromLocalStorage('store') || [];
+
```
Comment (mi-chal) [inline src/store/store.js]: Remove

File: src/store/store.js [hunk 4]
```diff
@@ -1,7 +1,22 @@
...
 const store = createStore(rootReducer, composeWithDevTools());
 
+// store.subscribe(
+//   throttle(() => {
+//     saveItemInLocalStorage('store', store.getState());
+//   }, 1000),
+// );
+
```
Comment (mi-chal) [inline src/store/store.js]: Remove

File: package.json [hunk 5]
```diff
@@ -12,6 +12,9 @@
     "@testing-library/user-event": "^12.6.0",
     "eslint-config-prettier": "^7.1.0",
     "eslint-plugin-prettier": "^3.3.1",
+    "loadash": "^1.0.0",
+    "loadsh": "0.0.4",
+    "lodash.throttle": "^4.1.1",
```
Comment (mi-chal) [inline package.json]: Remove dependencies

File: src/screens/Gift/styles.scss (hunk 1 above)
Comment (dwantula) [reply src/screens/Gift/styles.scss]: Done

File: src/store/store.js (hunk 2 above)
Comment (dwantula) [reply src/store/store.js]: Done

File: src/store/store.js (hunk 3 above)
Comment (dwantula) [reply src/store/store.js]: Done

File: src/store/store.js (hunk 4 above)
Comment (dwantula) [reply src/store/store.js]: Done

File: package.json (hunk 5 above)
Comment (dwantula) [reply package.json]: Done

**Pull request merged**